zomes/hc_zome_trust_atom/src/trust_atom.rs

```diff
@@ -48,7 +49,8 @@ pub fn create(
   target: EntryHash,
   content: &str,
   value: &str,
-  attributes: BTreeMap<String, String>,
+  bucket: Vec<u8>,
```

Comment:
I think bucket need not be passed in to create, can be created within it.

Reply:
fixed